### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by developertariq**
- Project created with webpack
- Used ES6 modules
- Followed wireframe guideline

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^7.1.3",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1"
```
Comment (GzimAsani) [inline package.json]: - [ ] webpack-dev-server should also be part of devDependencies on package.json, kindly install it

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (GzimAsani) [inline package.json]: - [ ] You need a way like `npm run` or `npm start` in order to start your app, see an example below:
image

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          <form id="entry-form">
+            <input type="text" id="name" name="name" placeholder="Your name" required><br>
+            <input type="number" id="score" name="score" placeholder="Your score" required><br><br>
+            <input type="submit" value="Submit">
+          </form>
+        </div>
+      </div>   
+    </section>      
+  </body>
+</html>
```
Comment (GzimAsani) [inline dist/index.html]: - [ ] Your [HTML](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), code should match the given wireframe, no styling is required at this stage

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @developertariq                             :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project structure :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 2 above)
Comment (developertariq) [reply package.json]: webpack-dev-server is configured

**Review by Gambit142 — APPROVED** [review]
Hi @developertariq 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Commit messages are capitalized and in imperative tense ✔️ 
- `Webpack` was set up properly ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**